dependencies.properties

```diff
@@ -226,10 +226,10 @@ org.crashub\:crash.embed.spring=1.3.0
 org.crashub\:crash.plugins.cron=1.3.0
 org.crashub\:crash.plugins.mail=1.3.0
 org.crashub\:crash.shell=1.3.0
-org.eclipse.jetty\:jetty-annotations=9.2.4.v20141103
-org.eclipse.jetty\:jetty-jsp=9.2.4.v20141103
-org.eclipse.jetty\:jetty-util=9.2.4.v20141103
-org.eclipse.jetty\:jetty-webapp=9.2.4.v20141103
+org.eclipse.jetty\:jetty-annotations=8.1.14.v20131031
```

Comment:
May be we can add a comment here mentioning jetty version needs to `8.1.14.v20131031` for spark streaming. Also, we need to investigate if this has any impact on other XD components.